colour/models/cie_xyy.py

```diff
@@ -134,6 +138,112 @@ def xyY_to_XYZ(xyY):
     return XYZ
 
 
+def xy_to_xyY(xy):
+    """
+    Converts from *xy* chromaticity coordinates to *CIE xyY* colourspace.
+
+    `xy` argument with last dimension being equal to 3 will be assumed to be a
```

Comment:
Sounds good!
Maybe also add the actual value used for Y.
Btw, could the Y actually be an optional value of the method with default 1?